### ISSUE TITLE
Show playlist for Sonarr

### ISSUE THREAD

**Issue by octavioj** [ytdl-org/youtube-dl#31073]
There is a piece of software called sonarr_youtubedl that allows for automated download of series. But it does require that one specifies the playlist of the show. Is anyone familiar if shows on CBS and CW, for instance, if there is a way to get the playlist of the shows? Thank you in advance.

**Comment by dirkf** [comment]
I suppose you mean https://github.com/whatdaybob/sonarr_youtubedl?

From the blurb it appears that this is just yt-dl in a Docker container suitable for linking into the Sonarr application.

Neither the CBS nor the CWTV extractor supports playlists, although apparently the latter at least has playlist pages on its geo-blocked site.

So no.

If someone who cares wishes to analyse the respective sites and propose pull requests for playlist support, the answer might change.

**Comment by octavioj** [comment]
> I suppose you mean https://github.com/whatdaybob/sonarr_youtubedl?
> 
> From the blurb it appears that this is just yt-dl in a Docker container suitable for linking into the Sonarr application.
> 
> Neither the CBS nor the CWTV extractor supports playlists, although apparently the latter at least has playlist pages on its geo-blocked site.
> 
> So no.
> 
> If someone who cares wishes to analyse the respective sites and propose pull requests for playlist support, the answer might change.

Yes, that is precisely the container. I tried looking for a playlist somewhere on their sites but could not find one. Thank you for the quick answer.
